scrtipt.js

```diff
@@ -0,0 +1,7 @@
+function openNav() {
+    document.getElementById("slidemenu").style.width = "100%";
+}
+
+function closeNav() {
+    document.getElementById("slidemenu").style.width = "0";
+}
```

Comment:
- [ ] Good Job for Implementing  the previous code reviewer suggestion When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed and the mobile menu disappear. So Kindly fix it When I clicked on the mobile menu options nothings happen.

Reply:
https://www.loom.com/share/0b7aa2eb04024f9dabee3e8543cddf8d